lib/src/time-range-dialog.dart

```diff
@@ -120,39 +120,39 @@ showTimeRangePicker({
   final Widget dialog = Dialog(
       elevation: 12,
       child: TimeRangePicker(
-        start: start,
-        end: end,
-        disabledTime: disabledTime,
-        paintingStyle: paintingStyle,
-        onStartChange: onStartChange,
-        onEndChange: onEndChange,
-        fromText: fromText,
-        toText: toText,
-        interval: interval,
-        padding: padding,
-        strokeWidth: strokeWidth,
-        handlerRadius: handlerRadius,
-        strokeColor: strokeColor,
-        handlerColor: handlerColor,
-        selectedColor: selectedColor,
-        backgroundColor: backgroundColor,
-        disabledColor: disabledColor,
-        backgroundWidget: backgroundWidget,
-        ticks: ticks,
-        ticksLength: ticksLength,
-        ticksWidth: ticksWidth,
-        ticksOffset: ticksOffset,
-        ticksColor: ticksColor,
-        snap: snap,
-        labels: labels,
-        labelOffset: labelOffset,
-        rotateLabels: rotateLabels,
-        autoAdjustLabels: autoAdjustLabels,
-        labelStyle: labelStyle,
-        timeTextStyle: timeTextStyle,
-        activeTimeTextStyle: activeTimeTextStyle,
-        hideTimes: hideTimes,
-      ));
+          start: start,
+          end: end,
+          disabledTime: disabledTime,
+          paintingStyle: paintingStyle,
+          onStartChange: onStartChange,
+          onEndChange: onEndChange,
+          fromText: fromText,
+          toText: toText,
+          interval: interval,
+          padding: padding,
+          strokeWidth: strokeWidth,
+          handlerRadius: handlerRadius,
+          strokeColor: strokeColor,
+          handlerColor: handlerColor,
+          selectedColor: selectedColor,
+          backgroundColor: backgroundColor,
+          disabledColor: disabledColor,
+          backgroundWidget: backgroundWidget,
+          ticks: ticks,
+          ticksLength: ticksLength,
+          ticksWidth: ticksWidth,
+          ticksOffset: ticksOffset,
+          ticksColor: ticksColor,
+          snap: snap,
+          labels: labels,
+          labelOffset: labelOffset,
+          rotateLabels: rotateLabels,
+          autoAdjustLabels: autoAdjustLabels,
+          labelStyle: labelStyle,
+          timeTextStyle: timeTextStyle,
+          activeTimeTextStyle: activeTimeTextStyle,
+          hideTimes: hideTimes,
+          use24HourFormat: use24HourFormat));
```

Comment:
was not sure if the default value is passed (true) or not, so I checked it here as well. 

If you disagree, please feel free to change this PR without getting back to me, it would be faster that way.